dto/vehicleDetailsDto.go

```
@@ -62,7 +61,7 @@ type VehicleSummary struct {
 func (dto *VehicleDetailsDto) ToModel() (*model.Vehicle, error) {
 	uuid, err := uuid.Parse(dto.Uuid)
 	if err != nil {
```

golangci-lint (1.64.8)

SA9003: empty branch

(staticcheck)